pkg/operator/starter.go

```diff
@@ -196,3 +200,27 @@ func isCustomCABundleUsed(kubeClient kubeclient.Interface) (bool, error) {
 	_, exists := cloudConfigCM.Data[caBundleKey]
 	return exists, nil
 }
+
+func withSecretHashAnnotation(kubeClient kubeclient.Interface, namespace, secret string) func(*appsv1.Deployment) error {
+	return func(deployment *appsv1.Deployment) error {
+		inputHashes, err := resourcehash.MultipleObjectHashStringMapForObjectReferences(
```

Comment:
Would it be possible to use a secret informer? `MultipleObjectHashStringMapForObjectReferenceFromLister` with an informer/lister from `kubeInformersForNamespaces`.

Reply:
Good catch. Done.